src/components/dashboard/PotentialBarriers.vue

```diff
@@ -1,7 +1,6 @@
 <template>
-  <div>
-    <div class="content">
-      <p>People with fewer resources have a harder time getting vaccinated.</p>
+  <div class="signal-grid-container">
```

Comment:
```suggestion
  <div class="signal-grid-container px-4">
```
with text-based content, it looks better when the padding matches the title/subtitle

Reply:
added the matching padding! thanks !